docs/next/components/Heading.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export function Heading({id = '', level = 1, children, className}) {
+  return React.createElement(
+    `h${level}`,
```

Comment:
Not really a fan of this kind of dynamic behavior. It can end up just making these components harder to find in code, and `React.createElement` is kind of confusing.

```
import cx from 'classnames';

interface Props {
  id?: string;
  level?: number;
  children: React.ReactNode;
  className: string;
}

export const Heading = ({id = '', level = 1, children, className): Props) => {
  const props = {
    id,
    className: cx(heading, className),
    children: (
      <>
        {children}
        <a ...>...</a>
      </>
    ),
  };

  switch (level) {
    case 1: return <h1 {...props} />;
    case 2: return <h2 {...props} />;
    // etc.
  }
}
```

Reply:
thanks! this was copied directly from their examples. updated.